### PR TITLE
AB-101 transaction details

### PR DESCRIPTION
**Pull request by mfalthaw**

### CODE REVIEW

File: src/views/pages/ProductTransactions/index.vue [hunk 1]
```diff
@@ -10,14 +10,37 @@
...
         <md-table-cell md-label="Date" md-sort-by="datetime">{{item.datetime}}</md-table-cell>
         <md-table-cell md-label="From" md-sort-by="from">{{item.from}}</md-table-cell>
         <md-table-cell md-label="To" md-sort-by="to">{{item.to}}</md-table-cell>
         <md-table-cell :md-label="`Amount (${item.productUnits})`" md-sort-by="amountOfProduct">{{item.amountOfProduct}}</md-table-cell>
         <md-table-cell :md-label="`Rate (${item.productUnits}/${item.currency})`" md-sort-by="costPerUnit">{{item.costPerUnit}}</md-table-cell>
       </md-table-row>
     </md-table>
+
+    <div class="view-dialog-wrapper">
+      <md-dialog v-if="selectedRow" :md-active.sync="showViewDialog">
```
Comment (enochtsang) [inline src/views/pages/ProductTransactions/index.vue]: Doesn't seem like the `v-if` does anything. I removed it and functionality looked the same.

File: src/views/pages/ProductTransactions/index.vue [hunk 2]
```diff
@@ -10,14 +10,37 @@
...
+              <md-icon>access_time</md-icon>
+              <span>Date: {{ selectedRow.datetime }}</span>
+            </div>
+            <h3><b>From</b></h3>
+            {{ selectedRow.from }}
+            <h3><b>To</b></h3>
+            {{ selectedRow.to }}
+            <h3><b>Amount ({{ selectedRow.productUnits }})</b></h3>            
+            {{ selectedRow.amountOfProduct }}
+            <h3><b>Rate ({{ selectedRow.productUnits }}/{{ selectedRow.currency }})</b></h3>
```
Comment (enochtsang) [inline src/views/pages/ProductTransactions/index.vue]: It's currency/productUnits

File: src/views/pages/ProductTransactions/index.vue (hunk 1 above)
Comment (mfalthaw) [reply src/views/pages/ProductTransactions/index.vue]: I think it's just a precaution if `selectedRow` was ever empty. Just like checking if object is null.

File: src/views/pages/ProductTransactions/index.vue (hunk 2 above)
Comment (mfalthaw) [reply src/views/pages/ProductTransactions/index.vue]: Nice catch, I copied what the table column said :p

Comment (enochtsang) [reply src/views/pages/ProductTransactions/index.vue]: Ah yea, can you fix that in this PR?

**Pull request merged**